tests/e2etest-xoss/main.k

```diff
@@ -0,0 +1,119 @@
+
+import models.io.upbound.dev.meta.v1alpha1 as metav1alpha1
+
+_items = [
+    metav1alpha1.E2ETest{
+        metadata.name: "e2etest-xoss"
+        spec= {
+            crossplane.autoUpgrade.channel: "Rapid"
+            defaultConditions: ["Ready"]
+            manifests: [
+                # Create an Oss claim to test the observability stack
+                {
+                    apiVersion: "observe.platform.upbound.io/v1alpha1"
+                    kind: "Oss"
+                    metadata = {
+                        name: "uptest-observability"
+                        namespace: "default"
+                    }
+                    spec.parameters = {
+                        id: "uptest-observability-oss-cluster"
+                        operators.prometheus.version: "52.1.0"
+                        deletionPolicy: "Delete"
+                    }
+                }
+            ]
+            extraResources: [
+                # AWS EKS Configuration
+                {
```

Comment:
Can we maybe use types for extra resources

Reply:
those are not direct dependencies so I don't have a models for them